candy-machine/program/tests/initialize.rs

```diff
@@ -85,6 +86,12 @@ async fn init_default_success() {
         .mint_and_assert_successful(context, Some(1), true)
         .await
         .unwrap();
+    let pre_balance = get_balance(context, &candy_manager.authority.pubkey()).await;
+    candy_manager.withdraw(context).await.unwrap();
+    let post_balance = get_balance(context, &candy_manager.authority.pubkey()).await;
+    assert_account_empty(context, &candy_manager.candy_machine.pubkey()).await;
+    assert_account_empty(context, &candy_manager.collection_info.pda).await;
+    assert!(post_balance > pre_balance);
```

Comment:
Since the lamports have to go somewhere, checking that both accounts are empty and that the authority balance is greater than it was before calling the instruction means the lamports must have increased by the entire balance of the accounts that are now empty.

Reply:
yeah makes sense